src/webgpu/api/validation/render_pass/resolve.spec.ts

```diff
@@ -129,17 +144,25 @@ Test various validation behaviors when a resolveTarget is provided.
             usage: resolveTargetUsage,
           });
 
-          renderPassColorAttachmentDescriptors.push({
-            view: resolveSourceColorAttachment.createView(),
-            loadOp: 'load',
-            storeOp: 'discard',
-            resolveTarget: resolveTarget.createView({
+          let resolveTargetView: GPUTextureView | undefined;
+
+          const shouldError =
+            resolveTargetViewMipCount === 0 || resolveTargetViewArrayLayerCount === 0;
+          t.expectValidationError(() => {
+            resolveTargetView = resolveTarget.createView({
               dimension: resolveTargetViewArrayLayerCount === 1 ? '2d' : '2d-array',
               mipLevelCount: resolveTargetViewMipCount,
               arrayLayerCount: resolveTargetViewArrayLayerCount,
               baseMipLevel: resolveTargetViewBaseMipLevel,
               baseArrayLayer: resolveTargetViewBaseArrayLayer,
-            }),
+            });
+          }, shouldError);
```

Comment:
And then here instead of expectValidationError, just do `resolveTargetInvalid ? this.getErrorTextureView() : ...`.

(It would actually be better to add a new `createTextureViewWithState` in `ValidationTest`, similar to `createBufferWithState` where it uses the original descriptor and changes just one thing to make it invalid - but we don't have that now. If you want to add it, please feel free.)

Reply:
ok, I just do getErrorTextureView() here for now.